website_sale_require_legal/static/tests/tours/tour.js

```diff
@@ -2,12 +2,11 @@
  * Copyright 2023 Pilar Vargas <pilar.vargas@tecnativa.com>
  * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
 
-odoo.define("website_sale_require_legal.tour", function (require) {
-    "use strict";
-
-    var tour = require("web_tour.tour");
-
-    var steps = [
+import {registry} from "@web/core/registry";
```

Comment:
this file should be suffixed as `.ems.js` in order to work as a js module and you should put `/** @odoo-module **/` in the first line for the transpiler to deal with it correctly

Reply:
Yes, thanks for the reminder